### PR TITLE
Install broken link checker node modules on staging machine.

### PR DESCRIPTION
**Pull request by breville**

### CODE REVIEW

File: lib/rake/build.rake [hunk 1]
```diff
@@ -21,6 +21,14 @@ namespace :build do
     end
   end
 
+  desc 'Builds broken link checker.'
```
Comment (Bjvanminnen) [inline lib/rake/build.rake:24]: Should desc be something like "builds a set of independent tools" and then have a comment above brokenLinkChecker that it's building this particular tool (i.e. presumably there may eventually be other tools we expect to live in here too?)

File: lib/rake/build.rake [hunk 2]
```diff
@@ -122,6 +130,7 @@ namespace :build do
   tasks << :apps if CDO.build_apps
   tasks << :dashboard if CDO.build_dashboard
   tasks << :pegasus if CDO.build_pegasus
+  tasks << :tools if rack_env?(:staging)
```
Comment (Bjvanminnen) [inline lib/rake/build.rake:133]: wonder if we should have a CDO.build_tools? @wjordan might have an opinion

**Comment by wjordan** [comment]
LGTM, though I do have two non-blocking concerns:

1. Is there a reason the entrypoint for this script needs to be in `tools/scripts`, rather than the `bin/cron` directory alongside all other cron jobs?
2. I worry about the precedent this sets for any small utility script to declare its dependencies with npm to need to include an additional task in `build.rake`. Fine for getting a one-off task going for now, but there's some technical debt here worth keeping in mind for future refactoring passes over our build structure.

**Comment by Bjvanminnen** [comment]
I wonder if as the number of utilities grows, it makes sense for them to all share a single package.json (and thus a single build step across all of them) instead of one per tool.

**Comment by breville** [comment]
Thanks @wjordan.  Regarding location, `tools/scripts` was suggested as a good place but am neutral on the issue.  Happy to move now or later if `bin/cron` deemed better.  @islemaster might have thoughts.

**Pull request merged**